src/styles.ts

```diff
@@ -103,4 +103,9 @@ const getStyles = (
 `;
 };
 
+export const addStylesForDiagram = (type: any, diagramTheme: any) => {
+  // @ts-ignore
```

Comment:
If you modify line 17 to `const themes: Record<string, any> = {`, you can remove the `@ts-ignore`.